pandas/io/sas/sas7bdat.py

```diff
@@ -687,6 +687,26 @@ def _read_next_page(self):
 
         return False
 
+    @staticmethod
+    def _convert_datetimes(sas_datetimes, unit):
+        """Converts to np.datetime64 if possible, otherwise to dateime.datetime
```

Comment:
we *never* convert to np.datetime64, no need to mention this, rather we convert to Timestamp (misspell on datetime)

Reply:
I've updated the docstring